### PR TITLE
Valve Landmark Model

### PR DESCRIPTION
**Pull request by ericspod**
Signed-off-by: Eric Kerfoot <eric.kerfoot@kcl.ac.uk>

### Description
Valve landmark regression model.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json`.
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [x] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [x] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by wyli** [comment]
for some reason the notebook has 100k lines, perhaps there are some hidden logging cells..

**Comment by ericspod** [comment]
It's from the stored video which is represented in the notebook as a base64 blob for the most part.

**Comment by wyli** [comment]
I see, perhaps move the notebook to the large file yaml? It'll make this codebase light weight.

**Comment by ericspod** [comment]
I've shrunk the notebook down to a single frame instead. It's better that there's something people can see in the repository.

**Comment by ericspod** [comment]
The output from blossom when running the bundle indicates the `PYTHONPATH` variable isn't being set to include the `scripts` directory. This was the place I had put script files in for the MedNIST GAN bundle but the CI/CD process I guess has changed since. We may want to update the specification to explicitly state a directory which will be added to the path when a bundle runs so the user also doesn't have to set `PYTHONPATH` on the command line. 

CI/CD output:
````
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,445 - INFO - Expected md5 is None, skip md5 check for file models/valve_landmarks/eval/schema.json.
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,458 - INFO - metadata is verified with no error.
[2022-09-02T09:32:55.728Z] metadata format is verified correctly.
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - --- input summary of monai.bundle.scripts.verify_net_in_out ---
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - > net_id: 'network_def'
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - > meta_file: 'models/valve_landmarks/configs/metadata.json'
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - > config_file: 'models/valve_landmarks/configs/inference.json'
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - > bundle_root: 'models/valve_landmarks'
[2022-09-02T09:32:55.728Z] 2022-09-02 09:32:55,459 - INFO - ---
[2022-09-02T09:32:55.728Z] 
[2022-09-02T09:32:55.728Z] 
[2022-09-02T09:32:55.728Z] 
[2022-09-02T09:32:55.728Z] Traceback (most recent call last):
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/config_item.py", line 285, in instantiate
[2022-09-02T09:32:55.728Z]     return instantiate(modname, **args)
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/utils/module.py", line 215, in instantiate
[2022-09-02T09:32:55.728Z]     raise ModuleNotFoundError(f"Cannot locate class or function path: '{path}'.")
[2022-09-02T09:32:55.728Z] ModuleNotFoundError: Cannot locate class or function path: 'valve_landmarks.PointRegressor'.
[2022-09-02T09:32:55.728Z] 
[2022-09-02T09:32:55.728Z] The above exception was the direct cause of the following exception:
[2022-09-02T09:32:55.728Z] 
[2022-09-02T09:32:55.728Z] Traceback (most recent call last):
[2022-09-02T09:32:55.728Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/ci/verify_bundle.py", line 175, in <module>
[2022-09-02T09:32:55.728Z]     verify(bundle)
[2022-09-02T09:32:55.728Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/ci/verify_bundle.py", line 162, in verify
[2022-09-02T09:32:55.728Z]     verify_data_shape(bundle_path, net_id, config_file)
[2022-09-02T09:32:55.728Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/ci/verify_bundle.py", line 98, in verify_data_shape
[2022-09-02T09:32:55.728Z]     verify_net_in_out(
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/scripts.py", line 506, in verify_net_in_out
[2022-09-02T09:32:55.728Z]     net = parser.get_parsed_content(key).to(device_)
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/config_parser.py", line 254, in get_parsed_content
[2022-09-02T09:32:55.728Z]     return self.ref_resolver.get_resolved_content(id=id, **kwargs)
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/reference_resolver.py", line 186, in get_resolved_content
[2022-09-02T09:32:55.728Z]     return self._resolve_one_item(id=id, **kwargs)
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/reference_resolver.py", line 164, in _resolve_one_item
[2022-09-02T09:32:55.728Z]     self.resolved_content[id] = item.instantiate() if kwargs.get("instantiate", True) else item
[2022-09-02T09:32:55.728Z]   File "/root/.local/share/virtualenvs/MONAI-zoo-premerge-CHR6GQKS/lib/python3.8/site-packages/monai/bundle/config_item.py", line 287, in instantiate
[2022-09-02T09:32:55.728Z]     raise RuntimeError(f"Failed to instantiate {self}.") from e
[2022-09-02T09:32:55.728Z] RuntimeError: Failed to instantiate {'_target_': 'valve_landmarks.PointRegressor', 'in_shape': [1, 256, 256], 'out_shape': [2, 10], 'channels': [8, 16, 32, 64, 128], 'strides': [2, 2, 2, 2, 2]}.
````

**Comment by yiheng-wang-nv** [comment]
Hi @ericspod , so far, the ci test adds the path of the bundle before verifying (see [here](https://github.com/Project-MONAI/model-zoo/blob/30b31f6f0c19d1f587f2037fd3f261a93bd1ea66/ci/verify_bundle.py#L144)), and within a bundle, it may need to include `scripts` (for example: https://github.com/Project-MONAI/model-zoo/blob/30b31f6f0c19d1f587f2037fd3f261a93bd1ea66/models/renalStructures_UNEST_segmentation/configs/inference.json#L12)

cc @wyli @Nic-Ma , do you think it is a suitable way?

**Comment by wyli** [comment]
thanks @yiheng-wang-nv, I think we can change the import statement, @ericspod please try this:
https://github.com/Project-MONAI/model-zoo/commit/478426fd84468f5d1ade6f023a7127740322eaad I couldn't modify your dev branch as it's the default branch

**Comment by ericspod** [comment]
That did it, thanks @yiheng-wang-nv @wyli !

**Comment by wyli** [comment]
perhaps a `__init__.py` is needed in the `scripts/` folder

**Comment by wyli** [comment]
this works a916b889e8936a9de207aaae9c9f14f550ebb4d7, but `ConvDenseBlock` is not available in v0.9.1, it's currently in 1.0.0rc1

File: models/valve_landmarks/configs/metadata.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                "channel_def": {
+                    "0": "image"
+                }
+            }
+        },
+        "outputs": {
+            "pred": {
+                "type": "tuples",
+                "format": "points",
+                "num_channels": 0,
```
Comment (wyli) [inline models/valve_landmarks/configs/metadata.json]:
```suggestion
                "num_channels": 2,
```

**Comment by ericspod** [comment]
When exporting to Torchscript the `key_in_ckpt` needs to be provided somehow since the saved state contains values for the network and logger object. I've removed the logger to solve the issue here.

**Review by wyli — APPROVED** [review]
thanks, all tests work fine now.

**Pull request merged**